frontend/src/components/CreatePairRoom/CreatePairRoom.tsx

```diff
@@ -0,0 +1,93 @@
+import { useRef } from 'react';
+
+import { useMutation } from '@tanstack/react-query';
+
+import Input from '@/components/common/Input/Input';
+import CompleteCreatePairRoomModal from '@/components/CreatePairRoom/CompleteCreatePairRoomModal';
+import CreatePairRoomModal from '@/components/CreatePairRoom/CreatePairRoomModal';
+
+import usePairNameInput from '@/hooks/pairRoomModal/usePairNameInputs';
+import useModal from '@/hooks/useModal';
+
+import { addPairName } from '@/apis/pairName';
+
+interface CreatePairRoomProps {
+  isOpen: boolean;
+  closeModal: () => void;
+}
+
+const CreatePairRoom = ({ isOpen, closeModal }: CreatePairRoomProps) => {
+  const roomCode = useRef('abcdef');
+
+  //query 는 폴더 구조를 맞추지 않아서 일단 여기에 뒀어요 !
+  const { mutate: addPairNameMutation } = useMutation({
+    mutationFn: addPairName,
+    onSuccess: (data) => {
+      roomCode.current = data;
+      openCompletePairRoomModal();
+      closeModal();
+    },
+    onError: (error) => {
+      alert(error.message);
+    },
+  });
+
+  const {
+    firstPairValidateOnChange,
+    secondPairValidateOnChange,
+    resetPairNameValue,
+    isButtonActive,
+    getPairNameValue,
+    firstPairValue,
+    secondPairValue,
```

Comment:
여기서 동작을 나타내는 `-onChange`, `-Value` 는 제거하는 게 어떤가요?  네이밍 시 함수의 동작을 한정하게 되면 함수의 동작 범위가 변경될 때 함수 이름도 함께 변경해야 하는 번거로움이 쉽게 발생할 것 같아요!

```suggestion
    handleFirstPair,
    handleSecondPair,
    resetPairNames,
    isButtonActive,
    getPairNames,
    firstPair,
    secondPair,
```